### ISSUE TITLE
semantic highlighting not applied to any first line containing require keyword

### ISSUE THREAD

**Issue by noornee** [ellisonleao/gruvbox.nvim#227]
**Describe the bug**
semantic tokens doesn't seem to apply to the first line of a lua file that begins with the `require` keyword.
I don't really understand how treesitter or semantic tokens work but this isn't normal

```
$ nvim --version
  NVIM v0.9.0-dev-1275+gcbbf8bd66
  Build type: Release
  LuaJIT 2.1.0-beta3
```

**To Reproduce**
1. Create a lua file i.e. `hello.lua`
2. require any module in the first and second line without assigning it to a local variable like so or (check screenshots):
```
require("hello")
require("world")
```
3. you'll see that the first line containing the `require` keyword doesn't apply semantic tokens when you inspect it i.e. `:Inspect`

**Expected behavior**
semantic highlighting is supposed to be applied to all lines containing the keyword. if the higlighting isn't going to be consistent, it should fallback to using treesitter ?

**Screenshots**
---

**Normal:** _not so normal because the semantic highlights isnt different from that of normal strings_
![comment](https://user-images.githubusercontent.com/71889751/227708419-2cb1c404-7b02-404c-97b8-1852a15d8ec8.png)

---

**Inspect the first line: no semantic highlighting**

![first](https://user-images.githubusercontent.com/71889751/227708431-fd1e2ad4-ad17-40ad-a3c3-4136af37a13a.png)

---

**Inspect the second line: semantic highlighting**

![second](https://user-images.githubusercontent.com/71889751/227708442-b2d40edd-c8b8-4030-a831-f1ed3c5326dd.png)

---

**Comment by noornee** [comment]
Hi, i'm sorry. I just realized that this issue isnt associated with the gruvbox colorscheme.
i tried it on another colorscheme and the issue still persists. What do you think is the cause? does it have to do with lua lsp?

**Comment by ellisonleao** [comment]
@noornee semantic highlights will be available on #211 when it's ready

**Comment by noornee** [comment]
> @noornee semantic highlights will be available on #211 when it's ready

Alrighttt, keep up the good work! :D
